shared/django_apps/reports/migrations/0018_testinstance_branch_testinstance_commitid.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.11 on 2024-04-24 18:22
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
```

Comment:
[kinda nit] 
Typically we run `python manage.py sqlmigrate <APP_NAME> <MIGRATION_NUMBER>`
and paste the SQL output as comments inside the migration, just to have a better idea of the SQL that's gonna run for the migration